management/src/main/scala/akka/management/AkkaManagement.scala

```diff
@@ -32,6 +32,20 @@ object AkkaManagement extends ExtensionId[AkkaManagement] with ExtensionIdProvid
 }
 
 final class AkkaManagement(implicit system: ExtendedActorSystem) extends Extension {
+
+  ManifestInfo(system).checkSameVersion(productName = "Akka Management",
+    dependencies = List(
+      "akka-discovery-consul",
```

Comment:
Hmm now that the 'meat' of akka-discovery moved to akka itself, it seems a bit strange to keep the 'contrib' akka-discovery modules with akka-management. Perhaps we should more them to their own repo(s) at some point?

Reply:
Perhaps, but as long as they are released together I don't think we should complicate it and allow different versions.